### ISSUE TITLE
Is there any plan support On-Device Training for LLM Lora ?

### ISSUE THREAD

**Issue by huanyingjun** [google-ai-edge/LiteRT#5249]
Dear
Is there any plan support On-Device Training for LLM Lora ?

Thanks

**Comment by sourcelite** [comment]
Hi @huanyingjun, currently loading of pre-trained Lora adapters is supported. 

There is no plan announcement to support the ondevice LLM lora training.  Reason is I think the constraint of training a llm even with LORA requires storing of gradients and some optimser states, this will need memory exceeding the current limits of standard mobile devices NPUs/GPUs. Thanks.
